pom.xml

```diff
@@ -1,330 +1,312 @@
-									<goal>prepare-agent-integration</goal>
-								</goals>
-							</execution>
-							<execution>
-								<id>jacoco-site</id>
-								<phase>verify</phase>
-								<goals>
-									<goal>report</goal>
-								</goals>
-							</execution>
-						</executions>
-					</plugin>
-				</plugins>
-			</build>
-		</profile>
-
-		<profile>
-			<id>test-h2</id>
-
-			<properties>
-				<database-test.active>true</database-test.active>
-				<database-test.hibernate.dialect>org.hibernate.dialect.H2Dialect</database-test.hibernate.dialect>
-				<database-test.url>jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1</database-test.url>
-				<database-test.username>sa</database-test.username>
-				<database-test.password>sa</database-test.password>
-			</properties>
-
-			<dependencyManagement>
-				<dependencies>
-					<dependency>
-						<groupId>com.h2database</groupId>
-						<artifactId>h2</artifactId>
-						<version>${version.h2}</version>
-					</dependency>
-				</dependencies>
-			</dependencyManagement>
-		</profile>
-
-		<profile>
-			<id>h2</id>
-
-			<properties>
-				<database.driver>org.h2.Driver</database.driver>
-				<database.hibernate.dialect>org.hibernate.dialect.H2Dialect</database.hibernate.dialect>
-			</properties>
-
-			<dependencyManagement>
-				<dependencies>
-					<dependency>
-						<groupId>com.h2database</groupId>
-						<artifactId>h2</artifactId>
-						<version>${version.h2}</version>
-					</dependency>
-				</dependencies>
-			</dependencyManagement>
-		</profile>
-	</profiles>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>nanithing</groupId>
+  <artifactId>nanithing-parent</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+  <name>Nanithing: parent</name>
+  <description>Nanithing: Parent</description>
+  <modules>
+    <module>nanithing-model</module>
+    <module>nanithing-service-interfaces</module>
+    <module>nanithing-web</module>
+    <module>nanithing-dao-interfaces</module>
+    <module>nanithing-services</module>
+    <module>nanithing-types</module>
+    <module>nanithing-jaxrs-interfaces</module>
+    <module>nanithing-dao-jpa</module>
+    <module>nanithing-migrations-sql</module>
+  </modules>
+  <properties>
+    <version.liquibase>3.8.0</version.liquibase>
+    <version.junit.jupiter>5.5.1</version.junit.jupiter>
+    <version.java>11</version.java>
+    <version.mockito>3.0.0</version.mockito>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <version.h2>1.4.199</version.h2>
+    <version.resteasy-test>4.3.0.Final</version.resteasy-test>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <db.env>dev</db.env>
+    <version.quarkus>0.23.2</version.quarkus>
+  </properties>
+  <dependencyManagement>
```

Comment:
Add the dependency to `microprofile-openapi-api` here, with version from `quarkus-bom` and scope provided!